dataactvalidator/validation_handlers/validator.py

```diff
@@ -191,6 +192,9 @@ def cross_validate_sql(rules, submission_id, short_to_long_dict, first_file, sec
                                 str(rule.rule_error_message), values, row['row_number'], str(rule.rule_label),
                                 rule.file_id, rule.target_file_id, rule.rule_severity_id])
 
+        logger.info('VALIDATOR_INFO: Completed SQL cross validation query %s on submission %s',
```

Comment:
This log's format may need to be updated based on the results of the comment above.